scripts/west_commands/completion/west-completion.fish

```diff
@@ -195,15 +195,141 @@ function __zephyr_west_complete_help
     end
 end
 
+function __zephyr_west_topdir
+    set -l cwd (pwd)
+    set -l fallback 1
+
+    if test (count $argv) -eq 2
+        set cwd $argv[1]
+        set fallback $argv[2]
+    end
+
+    set -l cwd_split (string split '/' $cwd)
+
+    while true
+        set -l tmp_path (path normalize /(string join "/" $cwd_split))
```

Comment:
late observation.

The fish version shipped with Ubuntu 22.04 is 3.3 (and 20.04 ships with 3.1), whereas `path` as built-in was introduced in fish v3.5.
https://fishshell.com/docs/current/relnotes.html#fish-3-5-0-released-june-16-2022

Zephyr supports Ubuntu 20.04 LTS and 22.04 LTS: https://docs.zephyrproject.org/latest/develop/getting_started/index.html#select-and-update-os
> This guide covers Ubuntu version 20.04 LTS and later.

https://packages.ubuntu.com/focal/fish
https://packages.ubuntu.com/jammy/fish

This means some strange errors for fish users on those older releases installing the completion handler.

Reply:
Good catch. This is indeed bad UX, I'll make a PR in the coming weeks to address that issue :slightly_smiling_face: